feature/experimental/gribi/otg_tests/dut_daemon_failure/dut_daemon_failure_test.go

```diff
@@ -0,0 +1,395 @@
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/gribi"
+	"github.com/openconfig/featureprofiles/internal/otgutils"
+	gnps "github.com/openconfig/gnoi/system"
+	grps "github.com/openconfig/gribi/v1/proto/service"
+	"github.com/openconfig/gribigo/fluent"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Settings for configuring the baseline testbed with the test
+// topology.
+//
+// The testbed consists of:
+//   ate:port1 -> dut:port1 and
+//   dut:port2 -> ate:port2
+//
+//   * ate:port1 -> dut:port1 subnet 192.0.2.0/30
+//   * ate:port2 -> dut:port2 subnet 192.0.2.4/30
+//
+//   * Destination network: -> 203.0.113.0/24
+
+const (
+	ipv4PrefixLen    = 30
+	ateDstNetCIDR    = "203.0.113.0/24"
+	ateDstNetStartIp = "203.0.113.1"
+	nhIndex          = 1
+	nhgIndex         = 42
+)
+
+var (
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		MAC:     "02:00:01:01:01:01",
+		IPv4:    "192.0.2.2",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	dutPort2 = attrs.Attributes{
+		Desc:    "dutPort2",
+		IPv4:    "192.0.2.5",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	atePort2 = attrs.Attributes{
+		Name:    "atePort2",
+		MAC:     "02:00:02:01:01:01",
+		IPv4:    "192.0.2.6",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	gRIBIDaemons = map[ondatra.Vendor]string{
+		ondatra.ARISTA:  "Gribi",
+		ondatra.CISCO:   "emsd",
+		ondatra.JUNIPER: "rpd",
+	}
+)
+
+// configInterfaceDUT configures the DUT interfaces.
+func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes) *oc.Interface {
+	i.Description = ygot.String(a.Desc)
+	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	if *deviations.InterfaceEnabled {
+		i.Enabled = ygot.Bool(true)
+	}
+
+	s := i.GetOrCreateSubinterface(0)
+	s4 := s.GetOrCreateIpv4()
+	if *deviations.InterfaceEnabled && !*deviations.IPv4MissingEnabled {
+		s4.Enabled = ygot.Bool(true)
+	}
+	s4a := s4.GetOrCreateAddress(a.IPv4)
+	s4a.PrefixLength = ygot.Uint8(ipv4PrefixLen)
+
+	return i
+}
+
+// configureDUT configures port1 and port2 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	d := gnmi.OC()
+
+	p1 := dut.Port(t, "port1")
+	i1 := &oc.Interface{Name: ygot.String(p1.Name())}
+	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), configInterfaceDUT(i1, &dutPort1))
+
+	p2 := dut.Port(t, "port2")
+	i2 := &oc.Interface{Name: ygot.String(p2.Name())}
+	gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), configInterfaceDUT(i2, &dutPort2))
```

Comment:
Please copy the ExplicitInterfaceinDefaultVRF and ExplicitPortSpeed deviations here

(Added to the ATE test in:
- https://github.com/openconfig/featureprofiles/commit/656ff4967fef66a0306e8fa8d416b570b9aafc9e#diff-54a6920d26a5dc282fb7809685af54ac4425e025a55084a0c793774875981186)
- https://github.com/openconfig/featureprofiles/commit/71695eafc3b6cdf0625b4a42b98f093d41b52d47#diff-54a6920d26a5dc282fb7809685af54ac4425e025a55084a0c793774875981186
)

Reply:
Deviations from ATE test added